Apps/Database/src/Migrations/GatewayDbContextModelSnapshot.cs

```diff
@@ -1384,6 +1384,126 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("Dependent", "gateway");
                 });
 
+            modelBuilder.Entity("HealthGateway.Database.Models.DependentAudit", b =>
+                {
+                    b.Property<Guid>("DependentAuditId")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("uuid");
+
+                    b.Property<string>("AgentUsername")
+                        .IsRequired()
+                        .HasMaxLength(255)
+                        .HasColumnType("character varying(255)");
+
+                    b.Property<string>("CreatedBy")
+                        .IsRequired()
+                        .HasMaxLength(60)
+                        .HasColumnType("character varying(60)");
+
+                    b.Property<DateTime>("CreatedDateTime")
+                        .HasColumnType("timestamp with time zone");
+
+                    b.Property<string>("DelegateHdId")
+                        .IsRequired()
```

Comment:
If we're allowing nullable values this should not be `IsRequired()`

Reply:
Table is standalone now.